### PR TITLE
Added my Contribution

### PR DESCRIPTION
**Pull request by prabaljainn**

### CODE REVIEW

File: content/participant/Jatinkumar.md [hunk 1]
```diff
@@ -1,5 +1 @@
----
-name: Jatin Kumar
```
Comment (CodingSpecies) [inline content/participant/Jatinkumar.md:2]: Why is this file being edited @prabaljainn ?

Comment (prabaljainn) [reply content/participant/Jatinkumar.md:2]: hey, you got me wrong, actually i Duplicated Jatin Kumar's file and re edited it @CodingSpecies

Comment (CodingSpecies) [reply content/participant/Jatinkumar.md:2]: @prabaljainn it shows here that is has been completely deleted though, and there is only one line

Comment (CodingSpecies) [reply content/participant/Jatinkumar.md:2]: Did you try to just copy it and then use it in yours, your fileis ok, but there's only one line in the other one?

Comment (prabaljainn) [reply content/participant/Jatinkumar.md:2]: Sorry if i have done anything wrong how to undo this?

**Review by CodingSpecies — COMMENTED** [review]
```
---
name: Jatin Kumar
institution/company: Mahatma Jyotiba Phule Rohilkhand University
github:Jatinkumar30
---
```

paste the above back into the Jatinkumar.md file. Then it should be fine @prabaljainn

**Comment by prabaljainn** [comment]
Hey @CodingSpecies Just did everything right 
image

**Review by CodingSpecies — APPROVED** [review]
Thank you for fixing this @prabaljainn 

## Thank you for contributing! This PR is successfully part of Hacktoberfest! 🎊

**Comment by prabaljainn** [comment]
Can i do something else ? I want 3 more Accepted PRs

**Comment by CodingSpecies** [comment]
@prabaljainn I think that if you would like to improve the documentation of this repo, I think we need a `CONTRIBUTNG.md` file to explain exactly what to do.

But, in order for your pull requests to be hacktoberfest accepted, make sure they are not minor changes, or anything not useful, because that will just leave an unvalid label. See the hacktoberfest documentation for more: https://hacktoberfest.digitalocean.com/
thanks

**Comment by prabaljainn** [comment]
Ok done I'll Update Readme.md file for further clarification. I can also make a short video to tell how to do PRs and all for other students.

**Pull request merged**